recipes/prismatic/meta.yaml

```diff
@@ -0,0 +1,167 @@
+        - {{ pin_subpackage('prismatic_gui', max_pin='x.x') }}
+        - {{ pin_subpackage('pyprismatic', max_pin='x.x') }}  # [not osx]
+    test:
+      source_files:
+        - SI100.XYZ
+      commands:
+        - prismatic -i SI100.XYZ
+        - command -v prismatic-gui  # [linux]
+        - test -f $PREFIX/Applications/prismatic-gui.app/Contents/MacOS/prismatic-gui  # [osx]
+        - python -c "import pyprismatic; pyprismatic.demo()"  # [not osx]
+
+
+
+  - name: prismatic_cli
+    build:
+      skip: True  # [win]
+      string: "{{processor}}_h{{ PKG_HASH }}_{{ build }}"
+      script: bash ${RECIPE_DIR}/combined_build_install.sh cli  # [not win]
+
+    requirements:
+      build:
+        - {{ compiler('c') }}                
+        - {{ compiler('cxx') }}              
+        - {{ cdt('libxext-devel') }}         # [linux]
+        - {{ cdt('mesa-libgl-devel') }}      # [linux]
+        - {{ cdt('mesa-dri-drivers') }}      # [linux]
+        - {{ cdt('libxau-devel') }}          # [linux]
+        - {{ cdt('libselinux-devel') }}      # [linux]
+        - {{ cdt('libxdamage') }}            # [linux]
+        - {{ cdt('libxfixes') }}             # [linux]
+        - {{ cdt('libxxf86vm') }}            # [linux]
+        - make                               # [unix]
+        - cmake
+      host:
+        - boost
+        - hdf5
+        - fftw
+        - zlib
+      run:
+        - zlib
+        - hdf5
+        - fftw
+
+
+    test:
+      source_files:
+        - SI100.XYZ
+      commands:
+        - prismatic -i SI100.XYZ
+
+
+  - name: prismatic_gui
+    build:
+      skip: True  # [win]
+      string: "{{processor}}_h{{ PKG_HASH }}_{{ build }}"
+      script: bash ${RECIPE_DIR}/combined_build_install.sh gui  # [not win]
+
+    requirements:
+      build:
+        - {{ compiler('c') }}                
+        - {{ compiler('cxx') }}              
+        - {{ cdt('libxext-devel') }}         # [linux]
+        - {{ cdt('mesa-libgl-devel') }}      # [linux]
+        - {{ cdt('mesa-dri-drivers') }}      # [linux]
+        - {{ cdt('libxau-devel') }}          # [linux]
+        - {{ cdt('libselinux-devel') }}      # [linux]
+        - {{ cdt('libxdamage') }}            # [linux]
+        - {{ cdt('libxfixes') }}             # [linux]
+        - {{ cdt('libxxf86vm') }}            # [linux]
+        - make                               # [unix]
+        - cmake
+      host:
+        - boost
+        - hdf5
+        - fftw
+        - qt
+        - h5py
+        - zlib
+      run:
+        - zlib
+      
+    test:
+      commands:
+        - command -v prismatic-gui  # [linux]
+        - test -f $PREFIX/Applications/prismatic-gui.app/Contents/MacOS/prismatic-gui  # [osx]
+
+  - name: pyprismatic
+    build:
+      skip: True  # [win or py2k or osx]
+      string: "{{processor}}_h{{ PKG_HASH }}_{{ build }}"
+      script: bash ${RECIPE_DIR}/combined_build_install.sh py  # [not win]
+
+    requirements:
+      build:
+        - python
```

Comment:
`python` and `pip` should only be in `host`

Reply:
Just realised this is poorly written, I mean they should be in `host` and not `build`. You probably still need `python` to be in `run` as well.

Reply:
No worries - I got the gist of what you meant, and have done exactly this in my latest commit.